client/lib/media/library-selected-store.js

```diff
@@ -13,24 +13,25 @@ var MediaStore = require( './store' ),
 /**
  * Module variables
  */
-var _media = {},
```

Comment:
Maybe consider exporting these private variables instead if we need them exposed for testing. It should keep the changeset down. 

Not a blocker though for this PR.